tools/accuracy_checker/accuracy_checker/adapters/detection.py

```diff
@@ -639,3 +639,283 @@ def process(self, raw, identifiers=None, frame_meta=None):
             )
 
         return result
+
+class PersonVehicleDetectionAdapter(Adapter):
+    __provider__ = 'person_vehicle_detection'
+    predcition_types = (DetectionPrediction, )
+    class_threshold_ = [0.0, 0.60, 0.85, 0.90, 0.70, 0.75, 0.80, 0.75]
+    kMinObjectSize = 25
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'iou_threshold': NumberField(
+                value_type=float, min_value=0, max_value=1, default=0.45, optional=True,
+                description='Iou threshold for NMS'),
+        })
+        return parameters
+
+    def configure(self):
+        self.iou_threshold = self.get_value_from_config('iou_threshold')
+
+    def process(self, raw, identifiers=None, frame_meta=None):
+        result = []
+        if isinstance(raw, dict):
+            bbox_pred = raw['bbox_pred']
+            proposals = raw['proposals']
+            cls_score = raw['cls_score']
+            img_height, img_width, _ = frame_meta[0]['image_size']
+            props_map = self.output_to_proposals(bbox_pred, proposals, cls_score, img_width, img_height)
+            pred_items = self.get_proposals(props_map)
+            result.append(pred_items)
+        else:
+            for batch_index in range(len(identifiers)):
+                bbox_pred = raw[batch_index]['bbox_pred']
+                proposals = raw[batch_index]['proposals']
+                cls_score = raw[batch_index]['cls_score']
+                img_height, img_width, _ = frame_meta[batch_index]['image_size']
+                props_map = self.output_to_proposals(bbox_pred, proposals, cls_score, img_width, img_height)
+                pred_items = self.get_proposals(props_map)
+                result.append(pred_items)
+        return result
+
+    def output_to_proposals(self, bbox_pred, proposals, cls_score, img_width, img_height):
+        ww = img_width
+        hh = img_height
+        sx = 1.0 / (960 / img_width)
```

Comment:
if I right understand 960 and 540, it is width and height from input shape, is not it?
Could not they be given from input shape (also provided inside frame meta like image size)

Reply:
Yes, I will generalize this